website_animate/static/src/js/o_animate.frontend.js

```diff
@@ -126,6 +126,19 @@ publicWidget.registry.WebsiteAnimate = publicWidget.Widget.extend({
 
         return this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    destroy: function () {
+        this._super.apply(this, arguments);
+        this.$target.find('.o_animate')
+            .removeClass('o_animating o_animated o_animate_preview')
+            .css({
+                'animation-name': '',
+                'animation-play-state': '',
+                'visibility': '',
+            });
+    },
```

Comment:
because the clean for save methos is called on snippet drop (in the page), I had to move the remove of the "o_animate_preview" class here. This class is specific to the edit mode but I didnt find an other solution

Reply:
It's fine removing it for now, it's a particular case. But could you not *add* it in the **widget** start too? instead of via the options? That way it would be consistent at least: in the widget start, you could try `if (editableMode) { ... add previewClass ... }` and then it's normal that it is removed in its `destroy`. And not mention in the editor option at all anymore.

Reply:
The widget is on the wrapwrap in the .js frontend so it won't work because the start is only called once.
We need to add the class "o_animate_preview" each time we add the class "o_animate" with the options.